benchmarks/rtb_cache_benchmarks.cpp

```diff
@@ -14,7 +14,7 @@ namespace po = boost::program_options;
 #include "../examples/bidder/ad.hpp"
 #include "../examples/bidder/geo.hpp"
 #include "../examples/bidder/geo_ad.hpp"
-#include "../examples/bidder/campaign_data.hpp"
+//#include "../examples/bidder/campaign_data.hpp"
```

Comment:
I would just drop the lines instead of commenting out, it's always possible to look back in GIT history when one needs to refresh memories, and code looks cleaner without commented out fragments.